docker-compose.yml

```diff
@@ -114,7 +114,8 @@ services:
       APP_STRATEGIES_PLAN: false
 
   san-api:
-    image: ghcr.io/ministryofjustice/hmpps-strengths-based-needs-assessments-api:latest
+    # TODO: API is temporarily pinned to a version without any Drug mappings
```

Comment:
What do we need to do to fix this "properly" in the API?

Reply:
we need to define the new data mappings for drugs